### PR TITLE
167 renaming items

### PR DESCRIPTION
**Pull request by hgutha**
Added a "What's this" text next to wherever a category or category group existed that opened a chakra ui alert explaining the difference categories and category groups.

### CODE REVIEW

**Review by Roshan-D — CHANGES_REQUESTED** [review]
I like it! Only thing I would say is just do away with the modal thing and have the alerts always visible just so it's easier for people to see when they are setting up expo in the future

File: src/components/projectGallery/ProjectCard.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React from "react";
 import { Card, Tag, Button } from "antd";
+import { Text, Alert, AlertIcon, AlertDescription, CloseButton, IconButton, Flex } from "@chakra-ui/react";
+// import { InfoIcon } from "@chakra-ui/icons";
```
Comment (Roshan-D) [inline src/components/projectGallery/ProjectCard.tsx:4]: remove this

File: src/components/admin/AdminContentList.tsx [hunk 2]
```diff
@@ -103,7 +112,41 @@ const AdminContentList: React.FC<Props> = props => {
...
+        {(props.title=="Categories" || props.title=="Category Groups") && 
+        (<Text pl={2} pt={2} pb={0} fontSize="xs" color="blue.500" _hover={{ cursor: "pointer", textDecoration: "underline" }} onClick={openAlert}>
+          What is this?
+        </Text>
+        )}
+      </Flex>
+        {isOpen && (props.title=="Categories") && (
+          <Alert status='info' variant='subtle' size='xs' mt={0} mb={2}>
+            <AlertIcon />
+            {/* <AlertTitle>Information</AlertTitle> */}
```
Comment (Roshan-D) [inline src/components/admin/AdminContentList.tsx:127]: get rid of all these other commented things also

**Review by Roshan-D — CHANGES_REQUESTED** [review]

File: src/components/projectGallery/ProjectCard.tsx [hunk 3]
```diff
@@ -44,7 +56,25 @@ const ProjectCard: React.FC<Props> = props => {
...
       )}
       {tags.map((tag: string) => (
-        <Tag>{tag}</Tag>
+        <>
+          <Flex>
+            <Tag>{tag}</Tag>
+            <Text fontSize="xs" color="blue.500" _hover={{ cursor: "pointer", textDecoration: "underline" }} onClick={() => handleComponentClick(tag)}>
+              What is this?
+            </Text>
+          </Flex>
```
Comment (ayush-goyal) [inline src/components/projectGallery/ProjectCard.tsx:65]: I think we should remove the "What is this" from this page. There are so many texts that it makes it distracting. I think having the explanation just on the admin page is enough and we don't need anything here.
Screenshot 2023-07-31 at 1 47 30 PM

File: src/components/projectStatus/ProjectTableContainer.tsx [hunk 4]
```diff
@@ -158,9 +171,25 @@ const ProjectTableContainer: React.FC<Props> = props => {
...
                 <>
-                  <Title level={5} key={category.id}>
-                    {category.name}
-                  </Title>
+                  <Flex>
+                    <Title level={5} key={category.id}>
+                      {category.name} 
+                    </Title>
+                    <Text pl={2} pt={1} pb={0} fontSize="xs" color="blue.500" _hover={{ cursor: "pointer", textDecoration: "underline" }} onClick={() => handleComponentClick(project.id, category.name)}>
+                      What is this?
```
Comment (ayush-goyal) [inline src/components/projectStatus/ProjectTableContainer.tsx:179]: Same for the rest of the "What is this" 
Screenshot 2023-07-31 at 1 48 23 PM

File: src/components/admin/AdminContentList.tsx [hunk 5]
```diff
@@ -103,7 +112,26 @@ const AdminContentList: React.FC<Props> = props => {
...
+        <Title level={3}>{props.title}</Title>
+      </Flex>
+
+      {(props.title=="Categories") && (
+        <Text pb={2}>
+          Categories are prizes or awards that hackathon submissions can win. 
+          For example, “Best Overall”  or “T-Mobile Winner” or “Best Design”. Categories belong 
+          to category groups for judging organization purposes.
+        </Text>
+      )}
```
Comment (ayush-goyal) [inline src/components/admin/AdminContentList.tsx:125]: I think keeping this here is enough. Everything else would be a little too much.

**Review by ayush-goyal — CHANGES_REQUESTED** [review]

File: src/components/admin/AdminContentList.tsx [hunk 6]
```diff
@@ -38,6 +39,14 @@ const AdminContentList: React.FC<Props> = props => {
   const [searchText, setSearchText] = useState("");
   const [isJudging, setIsJudging] = useState<any>(undefined);
 
+  const [isOpen, setIsOpen] = useState(true);
```
Comment (Roshan-D) [inline src/components/admin/AdminContentList.tsx:42]: all of these can be removed since we are now just always showing the description.

**Review by Roshan-D — CHANGES_REQUESTED** [review]
A lot of old code can be removed, see my comment. Also, can you make the text a Chakra UI banner so it's harder to miss?

Screenshot 2023-08-11 at 10 46 12 AM

**Review by Roshan-D — APPROVED** [review]
nice!

**Pull request merged**